crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -2110,6 +2110,53 @@ cww......L
 ccccJJKKLL
 ENDMAP
 
+NAME:   dreamdust_from_the_deep
+TAGS:   no_monster_gen no_item_gen no_tide
+DEPTH:  Shoals, !Shoals:$
+ORIENT: northwest
+WEIGHT: 3
+KFEAT:  X = open_sea
+SUBST:  # = xxx~
+NSUBST: ~ = 8:z / 2:Z / *:w
+KFEAT:  wzZ = deep_water
+NSUBST: - = 6:1 / 2:2 / *:-
+KFEAT:  -12 = shallow_water
+TILE:   -12 = dngn_shallow_water_murky
+: set_feature_name("shallow_water", "some foul and slimy water")
```

Comment:
I would avoid redefining feature names here. This is something we try to reserve for portal maps where a change in theme is more self-contained and all encompassing (the entire level is different). In general, for these types of vaults that need to establish a theme but also need to integrate into the existing level, go for a "less is more" approach. Carefully chosen custom monsters and loot as well as possibly some use of custom tiles for walls can establish the theme well without using multiple custom descriptions that can confuse the player as to whether they have some in-game effect.